src/Itil_Project.php

```diff
@@ -133,7 +133,7 @@ public static function showForProject(Project $project)
 
         $canedit = $project->canEdit($ID);
 
-        /** @var CommonITILObject $itemtype */
+        /** @var CommonITILObject|string $itemtype */
```

Comment:
The correct type for PHPStan should be this:
```suggestion
        /** @var class-string<CommonITILObject> $itemtype */
```

I do not know how IDEs are handling it.